patches/api/0007-Add-manual-totem-of-undying-resurrection-API.patch

```diff
@@ -0,0 +1,56 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Bjarne Koll <lynxplay101@gmail.com>
+Date: Wed, 19 Jan 2022 23:29:55 +0100
+Subject: [PATCH] Add manual totem of undying resurrection API
+
+This patch introduces new API methods to the living entity that allow
+plugins to manually trigger a totem of undying resurrection on any given
+living entity.
+
+This is specifically useful for plugins that use the #setHealth methods
+to manipulate the entities health.
+
+The API is a full representation of the server internal and fully
+simulates a resurrection including bukkit event calls and potion
+effects.
+
+diff --git a/src/main/java/org/bukkit/entity/LivingEntity.java b/src/main/java/org/bukkit/entity/LivingEntity.java
+index 31353bd20404a8c2acf6bf0df524dc3cae324272..f218d3623b82e6cd68530d34da12e0842dcfaa8f 100644
+--- a/src/main/java/org/bukkit/entity/LivingEntity.java
++++ b/src/main/java/org/bukkit/entity/LivingEntity.java
+@@ -910,4 +910,35 @@ public interface LivingEntity extends Attributable, Damageable, ProjectileSource
+      */
+     void setHurtDirection(float hurtDirection);
+     // Paper end
++
++    // KTP start
++    /**
++     * Attempts to resurrect this living entity with a totem of undying, regardless of currently health.
```

Comment:
Imagine adding an extraneous suffix to the beautiful word that was once "current." Absolutely ruined this entire PR due to the mistreatment of the Java docs. Fix yerself immediately!!! 😤